tests/test_e2e.py

```diff
@@ -535,20 +539,23 @@ def test_b2c_acquire_token_by_auth_code(self):
                 # This won't work https://msidlab.com/api/user?usertype=b2c
             password="***"  # From https://aka.ms/GetLabUserSecret?Secret=msidlabb2c
         """
+        config = self.get_lab_app_object(azureenvironment="azureb2ccloud")
         self._test_acquire_token_by_auth_code(
             authority=self._build_b2c_authority("B2C_1_SignInPolicy"),
-            client_id="b876a048-55a5-4fc5-9403-f5d90cb1c852",
+            client_id=config["appId"],
             port=3843,  # Lab defines 4 of them: [3843, 4584, 4843, 60000]
-            scope=["https://msidlabb2c.onmicrosoft.com/msaapp/user_impersonation"]
+            scope=config["defaultScopes"].split(',')  # Default scopes returned by lab are
```

Comment:
Why do we want to document the exact scopes return by lab here? Is that important?

Reply:
You are right, not really needed, removed them in the next commit

Reply:
Thanks for making the change. With the comments now gone, the missing of a trailing comma comes into attention. Python happens to allow an extra comma following the last parameter of a function, and we can take advantage of that.